spec.bs

```diff
@@ -38,6 +38,8 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     text: height; url: embedded-content-other.html#attr-dim-height
 spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
   type: dfn
+    text: request; url: #concept-request
+    text: policy; url: #content-security-policy-object
```

Comment:
https://fetch.spec.whatwg.org/#content-security-policy-object doesn't seem to link to anything

Reply:
I believe this is dead code from when I was trying to put the entirety of the pre and post request checks in the spec.